Modules/Libs/QuestieLib.lua

```diff
@@ -680,8 +680,8 @@ function QuestieLib:TextWrap(line, prefix, combineTrailing, splitOnDot, desiredW
                 if (string.sub(useLine, endIndex, endIndex) == " ") then
                     lastSpaceIndex = endIndex
                     --Track the dot at the end of a line
-                elseif (string.sub(useLine, endIndex, endIndex) == "." and endIndex ~= strlen(useLine) and splitOnDot) then
-                    dotIndex = endIndex
+                --elseif (string.sub(useLine, endIndex, endIndex) == "." and endIndex ~= strlen(useLine) and splitOnDot) then
```

Comment:
`splitOnDot` can most likely be removed entirely now.

And so can `dotIndex` on line 674.